docs/user-guide/cli.md

```diff
@@ -96,6 +96,16 @@ If you're using both these special comments and autofixing, please run stylelint
 
 For CSS with standard syntax, stylelint will use [postcss-safe-parser](https://github.com/postcss/postcss-safe-parser) to fix syntax errors.
 
+### Write Report to a File
+
+With the `--output-file filename` or `-o filename` option, stylelint will output the report to the specified `filename` in addition to the standard output.
```

Comment:
```suggestion
With the `--output-file filename` option, stylelint will output the report to the specified `filename` in addition to the standard output.
```

I think we don't need to add all aliases. Let's keep the more descriptive version only.